File: 3515-FindIfDigitGameCanBeWon/3515-FindIfDigitGameCanBeWon.js
// Last updated: 06/03/2026, 16:00:04
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var canAliceWin = function(nums) {
    let sumSingle = 0;
    let sumDouble = 0;

    for(let i = 0 ; i<nums.length ; i++){
        if(nums[i]<=9){
            sumSingle+=nums[i];
        }
        else{
            sumDouble+=nums[i];
        }
    }
    return sumSingle!==sumDouble;
};

File: 3515-FindIfDigitGameCanBeWon/3515-FindIfDigitGameCanBeWon.cpp
// Last updated: 17/06/2026, 11:59:05
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int sumSingle = 0;
        int sumDouble = 0;

        for (int i = 0; i < nums.size(); i++) {
            if (nums[i] <= 9) {
                sumSingle += nums[i];
            } else {
                sumDouble += nums[i];
            }
        }

        return sumSingle != sumDouble;
    }
};